projects.html

```diff
@@ -1,95 +1,160 @@
+    <!-- Robot.txt -->
+    <meta name="robots" content="index,follow" />
+    <meta name="Googlebot" content="index,follow" />
+    <meta name="AdsBot-Google" content="index,follow" />
 
-<head>
-  <title>John Doe | Projects</title>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta name="keywords"
-    content="free portfolio template,portfolio template, portfolio website template, open source and contribution to open source" />
-  <meta name="description"
-    content="This project serves as a free and beautiful portfolio template for everyone who needs the perfect portfolio. Anyone can showcase their work including moocs, education, projects, experience,etc. to the world." />
-  <!-- Google Analytics -->
-  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-167979537-2"></script>
-  <!-- Fetching our Google Tag Manager -->
-  <!-- Robot.txt -->
-  <meta name="robots" content="index,follow" />
-  <meta name="Googlebot" content="index,follow" />
-  <meta name="AdsBot-Google" content="index,follow" />
+    <meta
+      property="og:site_name"
+      content="https://portfolio.smaranjitghose.codes/projects.html"
+    />
+    <meta property="og:title" content="John Doe | Projects" />
+    <meta
+      property="og:description"
+      content="This project serves as a free and beautiful portfolio template for everyone who needs the perfect portfolio. Anyone can showcase their work including moocs, education, projects, experience,etc. to the world."
+    />
+    <meta property="og:type" content="website" />
+    <meta
+      property="og:url"
+      content="https://portfolio.smaranjitghose.codes/projects.html"
+    />
+    <meta name="twitter:card" content="summary" />
+    <meta
+      name="twitter:site"
+      content="https://portfolio.smaranjitghose.codes/projects.html"
+    />
 
-  <meta property="og:site_name" content="https://portfolio.smaranjitghose.codes/projects.html">
-  <meta property="og:title" content="John Doe | Projects">
-  <meta property="og:description"
-    content="This project serves as a free and beautiful portfolio template for everyone who needs the perfect portfolio. Anyone can showcase their work including moocs, education, projects, experience,etc. to the world.">
-  <meta property="og:type" content="website">
-  <meta property="og:url" content="https://portfolio.smaranjitghose.codes/projects.html">
-  <meta name="twitter:card" content="summary">
-  <meta name="twitter:site" content="https://portfolio.smaranjitghose.codes/projects.html">
+    <meta name="application-name" content="John Doe | Projects" />
+    <meta name="apple-mobile-web-app-title" content="John Doe | Projects" />
+    <meta
+      name="summary"
+      content="An open source project aimed at providing free and beautiful templates to everyone for building their portfolio websites and showcase their work to the world."
+    />
+    <link
+      rel="stylesheet"
```

Comment:
Avoid these indentation and formatting changes till the end else others will face merge conflicts😊

Reply:
Ok sure.
I'll take care of this